09.md

```diff
@@ -34,6 +34,31 @@ Relays SHOULD continue to publish/share the deletion request events indefinitely
 
 When an `a` tag is used, relays SHOULD delete all versions of the replaceable event up to the `created_at` timestamp of the deletion request event.
 
+## Filter Deletation
+
+A client MAY use `f` tag to provide a filter, relays and clients SHOULD consider deletation of all events matching to this specific filter from this client up to `kind 5` created at.
+
+For example:
+```jsonc
+{
+  "kind": 5,
+  "pubkey": <32-bytes hex-encoded public key of the event creator>,
+  "tags": [
+    ["f", "{\"kinds\":[1,7]}"]
```

Comment:
Is the author pubkey implicit here?

Reply:
the author field of course can only be set to event public key. maybe in groups and moderated communities, more public keys would be helpful...

Reply:
Why should relays honor deletion events from pubkeys that did not create those events in the first place? I believe having an implicit "authors" with the delete events pubkey makes more sense.

Reply:
most of times they won't. but in case if moderated communities or groups and similar stuff they MAY do it. but it can be implicit as well if more people think this one is better.

Reply:
just checked the moderated communities standard, my consideration doesn't make sense. I've updated the pr. thanks a lot.